.murdock

```diff
@@ -236,19 +244,33 @@ test_job() {
     local appdir=$1
     local board=$(echo $2 | cut -f 1 -d':')
     local toolchain=$(echo $2 | cut -f 2 -d':')
-    local flashfile="$3"
 
-    [ ! -f "$flashfile" ] && {
-        echo "$0: _test(): flashfile \"$flashfile\" doesn't exist!"
-        return 1
-    }
+    # interpret any extra arguments as file names.
+    # They will be sent along with the job to the test worker
+    # and stored in the application's binary folder.
+    shift 2
+    local files=""
+    for filename in "$@"; do
+        # check if the file is within $(BINDIR)
+        if startswith "${BINDIR}" "${filename}"; then
+          # get relative (to $(BINDIR) path
+          local relpath="$(realpath --relative-to ${BINDIR} ${filename})"
+        else
+          local relpath="$(basename ${filename})"
```

Comment:
If a file was in a subdirectory it ends up at the root of bindir. So not in the same place as the makefile would remotely expect it.

I do not see how this would work without a specific `murdock` hack in the test.

It should just be required or handled.

Reply:
This is "./,murdock". Everything in here is quite murdock specific. Yes, the SUIT makefile has a CI specific hack (if RIOT_CI_BUILD: KEYDIR=BINDIR, else: ...), but that is a pragmatic solution that works in practice.

Where exactly do you see an issue?

Reply:
`basename`.